incubator/exporter-prometheus/src/main/java/io/aklivity/zilla/runtime/exporter/prometheus/internal/PrometheusExporterContext.java

```diff
@@ -19,23 +19,27 @@
 import io.aklivity.zilla.runtime.engine.config.ExporterConfig;
 import io.aklivity.zilla.runtime.engine.exporter.ExporterContext;
 import io.aklivity.zilla.runtime.engine.exporter.ExporterHandler;
+import io.aklivity.zilla.runtime.exporter.prometheus.internal.config.PrometheusOptionsConfig;
 
 public class PrometheusExporterContext implements ExporterContext
 {
     private final EngineConfiguration config;
+    private final EngineContext context;
 
     public PrometheusExporterContext(
         EngineConfiguration config,
         EngineContext context)
     {
         this.config = config;
+        this.context = context;
     }
 
     @Override
     public ExporterHandler attach(
         ExporterConfig exporter)
     {
-        return new PrometheusExporterHandler(config, exporter);
+        PrometheusOptionsConfig options = (PrometheusOptionsConfig) exporter.options;
```

Comment:
Suggest creating a `PrometheusExporterConfig` with constructor `PrometheusExporterConfig(ExporterConfig)` and handle any type conversions in there, similar to `TlsBindingConfig`, etc.

Reply:
done